### PR TITLE
Restore GG Big Data Github

### PR DESCRIPTION
**Pull request by briane6873**
learning library ggbd git hub used in the Workshop has been corrupted. This is refreshing from a backup

### CODE REVIEW

**Comment by davidastart** [comment]
What is the current Workshop Management ID?

**Comment by briane6873** [comment]
Where do I find this ID?

Regards,
Brian R. Elliott
Oracle North America Sales
Data Integration Solutions
Mobile Phone:  +1.312.702.0040<tel:+13127020040>
Oracle Office: 233 S. Wacker Ste. Ste. 4500 Chicago IL 60606<x-apple-data-detectors://1/1>
***@***.***


From: ***@***.***>
Sent: Wednesday, July 28, 2021 8:57 AM
To: ***@***.***>
Cc: Brian ***@***.***>; ***@***.***>
Subject: [External] : Re: [oracle/learning-library] Restore GG Big Data Github (#3434)


What is the current Workshop Management ID?

**Comment by briane6873** [comment]
1780

Regards,
Brian R. Elliott
Oracle North America Sales
Data Integration Solutions
Mobile Phone:  +1.312.702.0040<tel:+13127020040>
Oracle Office: 233 S. Wacker Ste. Ste. 4500 Chicago IL 60606<x-apple-data-detectors://1/1>
***@***.***


From: ***@***.***>
Sent: Wednesday, July 28, 2021 8:57 AM
To: ***@***.***>
Cc: Brian ***@***.***>; ***@***.***>
Subject: [External] : Re: [oracle/learning-library] Restore GG Big Data Github (#3434)


What is the current Workshop Management ID?

File: data-management-library/goldengate/bigdata/mysql-to-cassandra/mysql-to-cassandra.md [hunk 1]
```diff
@@ -18,12 +18,13 @@ This lab assumes you have:
 - A Free Tier, Paid or LiveLabs Oracle Cloud account
 - SSH Private Key to access the host via SSH
 - You have completed:
-    - Lab: Generate SSH Keys (*Free-tier* and *Paid Tenants* only)
-    - Lab: Prepare Setup (*Free-tier* and *Paid Tenants* only)
```
Comment (rfontcha) [inline data-management-library/goldengate/bigdata/mysql-to-cassandra/mysql-to-cassandra.md:22]: Please revert

**Comment by briane6873** [comment]
reverted

Regards,
Brian R. Elliott
Oracle North America Sales
Data Integration Solutions
Mobile Phone:  +1.312.702.0040<tel:+13127020040>
Oracle Office: 233 S. Wacker Ste. Ste. 4500 Chicago IL 60606<x-apple-data-detectors://1/1>
***@***.***


From: Rene ***@***.***>
Sent: Wednesday, July 28, 2021 11:16 AM
To: ***@***.***>
Cc: Brian ***@***.***>; ***@***.***>
Subject: [External] : Re: [oracle/learning-library] Restore GG Big Data Github (#3434)


@rfontcha commented on this pull request.


In data-management-library/goldengate/bigdata/mysql-to-cassandra/mysql-to-cassandra.md<https://urldefense.com/v3/__https:/github.com/oracle/learning-library/pull/3434*discussion_r678456876__;Iw!!ACWV5N9M2RV99hQ!fw2DM815H3R0IuoQt-fXO6Nr1majI5Vafxs992pM4uoOVarFTwCoOSduu0ldaXyqwWZU$>:

> @@ -18,12 +18,13 @@ This lab assumes you have:

 - A Free Tier, Paid or LiveLabs Oracle Cloud account

 - SSH Private Key to access the host via SSH

 - You have completed:

-    - Lab: Generate SSH Keys (*Free-tier* and *Paid Tenants* only)

-    - Lab: Prepare Setup (*Free-tier* and *Paid Tenants* only)

Please revert

File: data-management-library/goldengate/bigdata/mysql-to-cassandra/mysql-to-cassandra.md [hunk 2]
```diff
@@ -15,13 +15,11 @@ In this lab we will load data in MySQL database `ggsource`, GG extract process `
...
-- SSH Private Key to access the host via SSH
-- You have completed:
-    - Lab: Generate SSH Keys
-    - Lab: Prepare Setup
-    - Lab: Environment Setup
-    - Lab: Deploy GoldenGate for Big Data
+ - A Free Tier, Paid or LiveLabs Oracle Cloud account
+ - SSH Private Key to access the host via SSH
+ - You have completed:
+ - Lab: Generate SSH Keys (*Free-tier* and *Paid Tenants* only)
```
Comment (rfontcha) [inline data-management-library/goldengate/bigdata/mysql-to-cassandra/mysql-to-cassandra.md:21]: 20-21 should be indented as they were before you deleted them. Please adjust

**Comment by briane6873** [comment]
Updated --

Regards,
Brian R. Elliott
Oracle North America Sales
Data Integration Solutions
Mobile Phone:  +1.312.702.0040<tel:+13127020040>
Oracle Office: 233 S. Wacker Ste. Ste. 4500 Chicago IL 60606<x-apple-data-detectors://1/1>
***@***.***


From: Rene ***@***.***>
Sent: Wednesday, July 28, 2021 4:03 PM
To: ***@***.***>
Cc: Brian ***@***.***>; ***@***.***>
Subject: [External] : Re: [oracle/learning-library] Restore GG Big Data Github (#3434)


@rfontcha commented on this pull request.


In data-management-library/goldengate/bigdata/mysql-to-cassandra/mysql-to-cassandra.md<https://urldefense.com/v3/__https:/github.com/oracle/learning-library/pull/3434*discussion_r678650317__;Iw!!ACWV5N9M2RV99hQ!YC3XLjdeomDVLQGimgEfdoIJv5nShg8dR1nX1tg5oxxBaUOLO0YpZ3joUv0yIbPvg2oQ$>:

> @@ -15,13 +15,11 @@ In this lab we will load data in MySQL database `ggsource`, GG extract process `



 ### Prerequisites

 This lab assumes you have:

-- A Free Tier, Paid or LiveLabs Oracle Cloud account

-- SSH Private Key to access the host via SSH

-- You have completed:

-    - Lab: Generate SSH Keys

-    - Lab: Prepare Setup

-    - Lab: Environment Setup

-    - Lab: Deploy GoldenGate for Big Data

+ - A Free Tier, Paid or LiveLabs Oracle Cloud account

+ - SSH Private Key to access the host via SSH

+ - You have completed:

+ - Lab: Generate SSH Keys (*Free-tier* and *Paid Tenants* only)

20-21 should be indented as they were before you deleted them. Please adjust

File: data-management-library/goldengate/bigdata-new/deploy-gg-for-bd/deploy-gg-for-bd.md [hunk 3]
```diff
@@ -1,23 +1,27 @@
...
-- Deploy GoldenGate for Big Data
+- Deploy GoldenGate for Big Data using Microservices
 
 ### Prerequisites
 This lab assumes you have:
 - A Free Tier, Paid or LiveLabs Oracle Cloud account
 - SSH Private Key to access the host via SSH
 - You have completed:
-    - Lab: Generate SSH Keys (*Free-tier* and *Paid Tenants* only)
-    - Lab: Prepare Setup (*Free-tier* and *Paid Tenants* only)
```
Comment (rfontcha) [inline data-management-library/goldengate/bigdata-new/deploy-gg-for-bd/deploy-gg-for-bd.md:20]: @briane6873  [19-20] should remain and indented as shown. Please remove [21-22]

File: data-management-library/goldengate/bigdata-new/deploy-gg-for-bd/deploy-gg-for-bd.md [hunk 4]
```diff
@@ -115,3 +121,9 @@ You may now *proceed to the next lab*.
 * **Author** - Brian Elliott, Data Integration Team, Oracle, August 2020
 * **Contributors** - Meghana Banka, Rene Fontcha
 * **Last Updated By/Date** - Rene Fontcha, Master Principal Solutions Architect, NA Technology, October 2020
+
+
+## Need Help?
+Please submit feedback or ask for help using our [LiveLabs Support Forum](https://community.oracle.com/tech/developers/categories/goldengate-on-premises). Please click the **Log In** button and login using your Oracle Account. Click the **Ask A Question** button to the left to start a *New Discussion* or *Ask a Question*.  Please include your workshop name and lab name.  You can also include screenshots and attach files.  Engage directly with the author of the workshop.
+
+If you do not have an Oracle Account, click [here](https://profile.oracle.com/myprofile/account/create-account.jspx) to create one.
```
Comment (rfontcha) [inline data-management-library/goldengate/bigdata-new/deploy-gg-for-bd/deploy-gg-for-bd.md:129]: The whole "Need Help?" section is no longer used. Please remove

File: data-management-library/goldengate/bigdata-new/mysql-to-cassandra/mysql-to-cassandra.md [hunk 5]
```diff
@@ -293,4 +294,9 @@ You may now *proceed to the next lab*.
 ## Acknowledgements
 * **Author** - Brian Elliott, Data Integration Team, Oracle, August 2020
 * **Contributors** - Meghana Banka, Rene Fontcha
-* **Last Updated By/Date** - Rene Fontcha, Master Principal Solutions Architect, NA Technology, October 2020
+* **Last Updated By/Date** -  Brian Elliott, Data Integration Team, Oracle, July 2021
+
+## Need Help?
+Please submit feedback or ask for help using our [LiveLabs Support Forum](https://community.oracle.com/tech/developers/categories/goldengate-on-premises). Please click the **Log In** button and login using your Oracle Account. Click the **Ask A Question** button to the left to start a *New Discussion* or *Ask a Question*.  Please include your workshop name and lab name.  You can also include screenshots and attach files.  Engage directly with the author of the workshop.
+
+If you do not have an Oracle Account, click [here](https://profile.oracle.com/myprofile/account/create-account.jspx) to create one.
```
Comment (rfontcha) [inline data-management-library/goldengate/bigdata-new/mysql-to-cassandra/mysql-to-cassandra.md:302]: Remove "Need Help?" from this and all MD's you changed

**Comment by briane6873** [comment]
Updated -

Regards,
Brian R. Elliott
Oracle North America Sales
Data Integration Solutions
Mobile Phone:  +1.312.702.0040<tel:+13127020040>
Oracle Office: 233 S. Wacker Ste. Ste. 4500 Chicago IL 60606<x-apple-data-detectors://1/1>
***@***.***


From: Rene ***@***.***>
Sent: Wednesday, July 28, 2021 6:09 PM
To: ***@***.***>
Cc: Brian ***@***.***>; ***@***.***>
Subject: [External] : Re: [oracle/learning-library] Restore GG Big Data Github (#3434)


@rfontcha commented on this pull request.


In data-management-library/goldengate/bigdata-new/deploy-gg-for-bd/deploy-gg-for-bd.md<https://urldefense.com/v3/__https:/github.com/oracle/learning-library/pull/3434*discussion_r678710792__;Iw!!ACWV5N9M2RV99hQ!f_Q6stzWlxDV32zaK0tP9rmnSA-RjCFKxUh-l4TlQrJYj_4A_1-zQ8U8pLXZspokBQb8$>:

>

 ### Prerequisites

 This lab assumes you have:

 - A Free Tier, Paid or LiveLabs Oracle Cloud account

 - SSH Private Key to access the host via SSH

 - You have completed:

-    - Lab: Generate SSH Keys (*Free-tier* and *Paid Tenants* only)

-    - Lab: Prepare Setup (*Free-tier* and *Paid Tenants* only)

@briane6873<https://urldefense.com/v3/__https:/github.com/briane6873__;!!ACWV5N9M2RV99hQ!f_Q6stzWlxDV32zaK0tP9rmnSA-RjCFKxUh-l4TlQrJYj_4A_1-zQ8U8pLXZssoPt4Wp$> [19-20] should remain and indented as shown. Please remove [21-22]

**Comment by rfontcha** [comment]
@briane6873 to resubmit

**Pull request closed without merging**